source/Esf.Website/App/esfState/esfState.component.html

```diff
@@ -1,66 +1,65 @@
-﻿<nav class="navbar navbar-default">
-    <div class="container">
-        <div class="navbar-header">
-            <a class="navbar-brand" href="#">ESFiddle</a>
-        </div>
-        <button type="button" 
-                (click)="runClicked()" 
-                [ngClass]="{'disabled' : !runCommandEnabled}"
-                class="run-command btn btn-default navbar-btn">
-            <span>Run</span>
-            <span *ngIf="runCommandInProgress">
-                <!--TODO: add spinner gif-->
-                ...
-            </span>
-        </button>
-
-        <button type="button" 
-                (click)="saveClicked()" 
-                [ngClass]="{'disabled' : !saveCommandEnabled}"
-                class="save-command btn btn-default navbar-btn">
+﻿<div fxFlexFill fxFlex fxLayout="column">
+    <md-toolbar color="primary" fxLayout="row">
+        <a class="brand" href="#">ESFiddle</a>
+        <button md-button
+                (click)="saveCommand.run(state)"
+                [disabled]="!saveCommandEnabled"
+                class="save-command">
+            <md-icon *ngIf="!saveCommandInProgress">save</md-icon>
+            <md-spinner *ngIf="saveCommandInProgress" color="accent"></md-spinner>
             <span>Save</span>
-            <span *ngIf="saveCommandInProgress">
-                <!--TODO: add spinner gif-->
-                ...
-            </span>
         </button>
+        <button md-button
+                (click)="runQueryCommand.run(state)" 
+                [disabled]="!runCommandEnabled"
+                class="run-command">
+            <md-icon *ngIf="!runCommandInProgress">play_arrow</md-icon>
+            <md-spinner *ngIf="runCommandInProgress" color="accent"></md-spinner>
+            <span>Run</span>
+        </button>
+    </md-toolbar>
+    <div fxLayout="row" fxFlex>
+        <vertical-split-view fxLayout="column" (onResize)="splitViewResized()" fxFlex>
+            <view1 fxLayout="column">
+                <horizontal-split-view (onResize)="splitViewResized()">
+                    <view1 fxLayout="column" fxFlex>
```

Comment:
We can improve the manes of view1 and view2 to reflect that they are top and bottom views